### ISSUE TITLE
Neopixel Support for AMLogic Boards specifically S905X

### ISSUE THREAD

**Issue by Iainrach** [adafruit/Adafruit_Blinka#652]
Trying to use the great Neopixel library on a Libre board but it seems like it is not supported. Any thoughts on providing this ?

It appears for the bcm283x boards you utilize the _rpi_ws281x library.

Does this mean it would be pretty difficult to get support for the Libre boards ?

**Comment by makermelissa** [comment]
Do you know of a ws2812 library that runs on the libre?

**Comment by Iainrach** [comment]
Not so far but I can drive a Max 7219 display over the SPI bus via Blinka
so perhaps this could work ??

Overview | CircuitPython NeoPixel Library Using SPI | Adafruit Learning
System
<https://learn.adafruit.com/circuitpython-neopixels-using-spi/overview>

**Comment by makermelissa** [comment]
Yeah, it's a good alternative. Yeah, if there's no existing library, it can get pretty difficult to add.

**Comment by makermelissa** [comment]
Also, I think PWM support is a requirement to add to other GPIO pins and most boards don't have it implemented and if they do, it often hasn't been added to Blinka.
